app/controllers/admin/schedules_controller.rb

```diff
@@ -31,14 +39,17 @@ def update
         error_message = "Could not find event GUID: #{params[:event]}"
       end
 
+      event_schedule = event.event_schedules.find_by(schedule_id: params[:schedule])
```

Comment:
For me this action looks more like it would belong to a EventSchedules Controller!
@hennevogel what do you think?
Also the :update action covers now :create, :update and :destroy ...